T/tblis/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "tblis"
+version = v"1.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/devinamatthews/tblis.git", "3e4c4b82943726c443b6f408c9c9791dcad7a847")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd tblis/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+cp lib/.libs/libtblis.so.0.0.0 /workspace/destdir/lib/libtblis.so.0
+cp src/external/tci/lib/.libs/libtci.so.0.0.0 /workspace/destdir/lib/libtci.so.0
```

Comment:
Isn't there a `make install` rule to install the libraries?  Better to use the [environment variables](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1) rather than the hardcoded paths:
```suggestion
cp lib/.libs/libtblis.so.0.0.0 ${libdir}/libtblis.so.0
cp src/external/tci/lib/.libs/libtci.so.0.0.0 ${libdir}/libtci.so.0
```

Reply:
There is, but it installs the `.la` files to `${libdir}` instead of the `.so` files which is not what i want to expose as a product.